app/src/main/java/kr/co/connect/boostcamp/livewhere/di/databaseModule.kt

```diff
@@ -0,0 +1,24 @@
+package kr.co.connect.boostcamp.livewhere.di
+
+import androidx.room.Room
+import kr.co.connect.boostcamp.livewhere.data.database.AppDataBase
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module.module
+
+const val DATABASE_NAME = "mdatabase.db"
```

Comment:
`p3;` prefix로 왜 m이 붙어야 하는지 잘 모르곘습니다.
```suggestion
const val DATABASE_NAME = "database.db"
```